sunpy/net/hek/util.py

```diff
@@ -0,0 +1,150 @@
+import os
+import re
+import json
+from pathlib import Path
+
+from regions import PolygonSkyRegion
+
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+from sunpy.time import parse_time
+
+UNIT_FILE_PATH = Path(os.path.dirname(__file__)) / "unit_properties.json"
+COORD_FILE_PATH = Path(os.path.dirname(__file__)) / "coord_properties.json"
+
+cm2 = u.def_unit("cm2", u.cm**3)
+m2 = u.def_unit("m2", u.m**2)
+m3 = u.def_unit("m3", u.m**3)
+
+u.add_enabled_units([cm2, m2, m3])
+
+u.set_enabled_aliases({
+    "steradian": u.sr,
+    "arcseconds": u.arcsec,
+    "degrees": u.deg,
+    "sec": u.s,
+    "Emx": u.Mx,
+    "Amperes": u.A,
+    "ergs": u.erg,
+    "cubic centimeter": u.ml,
+    "square centimeter": cm2,
+    "cubic meter": m3,
+    "square meter": m2,
+})
```

Comment:
So we want to update this to use the context manager that Jack suggested to register the units instead of doing this globally.